### PR TITLE
Fix: Add return objects for requests and correct return types in proto for notification service

### PR DESCRIPTION
**Pull request by ZixMai**
1.  Added return objects for requests.
2.  Corrected return types in proto for notification service.

### CODE REVIEW

File: notification_service/repository/mock_notification_repository.py [hunk 1]
```diff
@@ -263,18 +268,21 @@ async def create_notification(self, notification: Notification) -> None:
                 if self._notifications[i].enabled:
                     raise UniqueError("Notifications already exists")
                 self._notifications[i].enabled = True
+                notification = self._notifications[i]
```
Comment (Nhsdkk) [inline notification_service/repository/mock_notification_repository.py]: Use next instead of iterating

**Review by MaxPlays35 — APPROVED** [review]
Good

File: notification_service/repository/notification_repository_impl.py [hunk 2]
```diff
@@ -278,29 +278,33 @@ async def create_notification(self, notification: Notification) -> None:
         if db_notification is not None:
             if db_notification.enabled:
```
Comment (Nhsdkk) [inline notification_service/repository/notification_repository_impl.py:279]: The notification can be deleted and turned on. Make check for that

File: notification_service/repository/notification_repository_impl.py [hunk 3]
```diff
@@ -278,29 +278,33 @@ async def create_notification(self, notification: Notification) -> None:
...
+                        "author_id": notification.author_id,
+                    },
+                    data={"enabled": True}
+                    | (
+                        {"created_at": datetime.now(), "deleted_at": None}
+                        if db_notification.deleted_at is not None
+                        else {}
+                    ),
+                )
+                return await transaction.notification.find_first(
```
Comment (Nhsdkk) [inline notification_service/repository/notification_repository_impl.py]: Wrong return type

File: notification_service/repository/notification_repository_impl.py [hunk 4]
```diff
@@ -278,29 +278,33 @@ async def create_notification(self, notification: Notification) -> None:
...
                     where={
                         "event_id": notification.event_id,
                         "author_id": notification.author_id,
                     },
-                    data={"created_at": datetime.now(), "deleted_at": None},
                 )
         else:
-            await self._db_client.db.notification.create(
+            return await self._db_client.db.notification.create(
```
Comment (Nhsdkk) [inline notification_service/repository/notification_repository_impl.py]: Wrong return type

File: notification_service/repository/notification_repository_impl.py [hunk 5]
```diff
@@ -315,7 +319,7 @@ async def update_notification(self, notification: Notification) -> None:
             Catch all for every exception raised by Prisma Client Python.
 
         """
-        await self._db_client.db.notification.update(
+        return await self._db_client.db.notification.update(
```
Comment (Nhsdkk) [inline notification_service/repository/notification_repository_impl.py]: Wrong return type

**Review by Nhsdkk — CHANGES_REQUESTED** [review]
Make changes

File: notification_service/repository/notification_repository_impl.py (hunk 2 above)
Comment (ZixMai) [reply notification_service/repository/notification_repository_impl.py:279]: No, dear @Nhsdkk , it doesn't works like that 8)

**Review by Nhsdkk — APPROVED** [review]
LGTM

**Pull request merged**